src/components/FuncCode.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+import {CardText} from 'material-ui';
+
+class FuncCode extends Component {
```

Comment:
This looks like it could be a functional stateless component(FSC).  You wouldn't want to set props to state(that's like resetting state that's already set).  So it would just be something like: 
```
const FuncCode = (props) => (
     <CardText> props.code </CardText>
)

export default FuncCode
```

Reply:
I planned to highlight syntax with `prismjs` and I wasn't sure if I' gonna need it. I would leave it like this for now.